crates/iota-indexer/src/store/pg_indexer_store.rs

```diff
@@ -279,7 +291,80 @@ impl PgIndexerStore {
                 .first::<(i64, i64)>(conn)
                 .map(|(min, max)| (min as u64, max as u64))
         })
-        .context("Failed reading transaction range from PostgresDB")
+        .context(
+            format!("Failed reading transaction range from PostgresDB for checkpoint {checkpoint}")
```

Comment:
lowercase for error messages 

```suggestion
            format!("failed reading transaction range from PostgresDB for checkpoint {checkpoint}")
```